great_expectations/rule_based_profiler/estimators/exact_numeric_range_estimator.py

```diff
@@ -44,25 +44,22 @@ def _get_numeric_range_estimate(
         variables: Optional[ParameterContainer] = None,
         parameters: Optional[Dict[str, ParameterContainer]] = None,
     ) -> NumericRangeEstimationResult:
-        ndarray_is_datetime_type: bool
-        metric_values_converted: np.ndarray
+        datetime_detected: bool = datetime_semantic_domain_type(domain=domain)
+        metric_values_converted: np.ndaarray
         (
-            ndarray_is_datetime_type,
+            _,
+            _,
```

Comment:
@NathanFarmer They are not used in the present function -- only the converted array (which, since it is best effort, will be accepted with whatever `dtype` it will carry).  Thanks!